src/app/containers/CpsRelatedContent/index.jsx

```diff
@@ -63,6 +77,8 @@ const CpsRelatedContent = ({ content, enableGridWrapper }) => {
     children: node.isRequired,
   };
   if (!content.length) return null;
+  const hasSingleRelatedContent = content.length === 1;
+  const [singleRelatedContent] = content;
```

Comment:
is there another need for `singleRelatedContent` if we can just do `content[0]` on line 98 ?

Reply:
If we're going for readability, `singleRelatedContent` is more readable than `content[0]`.